src/test/java/org/microbean/jersey/netty/RequestTest.java

```diff
@@ -0,0 +1,74 @@
+package org.microbean.jersey.netty;
+
+import io.netty.bootstrap.ServerBootstrap;
+import io.netty.channel.ChannelFuture;
+import io.netty.channel.nio.NioEventLoopGroup;
+import io.netty.channel.socket.nio.NioServerSocketChannel;
+import org.glassfish.jersey.server.ApplicationHandler;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.net.InetSocketAddress;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class RequestTest {
+
+    private static NioEventLoopGroup group;
+    private static WebTarget webTarget;
```

Comment:
Minor formatting issue: please use two spaces throughout and separate variable declarations with a blank line.

Reply:
Sould I also add a PR for .editorconfig?

Reply:
Sure; hadn't heard of this file/project before.